README.md

```diff
@@ -142,52 +155,38 @@ The format of the JSON follows this typescript format: <src/utils/metadata.ts>
 
 To set your own `AppData`, change `REACT_APP_FULL_APP_DATA_<environment>` environment variable. For more details, check out the environment file (<.env>)
 
-### Supported networks
+## Orderbook API Endpoints
 
-You should set your own RPC endpoints by defining the following environment variables:
+Fee quote requests and posting orders are sent to the Orderbook API. This API has the responsibility of collecting orders and
+handing them to the solvers.
 
-```ini
-# Define your own RPC endpoints
-REACT_APP_NETWORK_URL_1=https://mainnet.infura.io/v3/{YOUR_INFURA_KEY}
-REACT_APP_NETWORK_URL_5=https://goerli.infura.io/v3/{YOUR_INFURA_KEY}
-REACT_APP_NETWORK_URL_100=https://rpc.gnosischain.com
-```
+The reference implementation of the API is [gp-v2-services](https://github.com/cowprotocol/services).
 
-If you plan to use Infura, you can omit defining the RPC endpoints, and just define the `REACT_APP_INFURA_KEY` environment variable.
+The API endpoint is configured using the environment variable `REACT_APP_ORDER_BOOK_URLS`.
+
+Normally, you don't need to change this variable, as it has a default value, but it can be handy to test your own API.
 
 ```ini
-# Alternatively you can just define your Infura key (instead of defining the RPC endpoints)
-REACT_APP_INFURA_KEY={YOUR_INFURA_KEY}
+REACT_APP_ORDER_BOOK_URLS='{"1":"https://api.cow.fi/mainnet","100":"https://api.cow.fi/xdai","5":"https://api.cow.fi/mainnet"}'
 ```
 
-````
+# Wallet Connect
 
-### API endpoints
-
-Fee quote requests and posting orders are sent to an API. This API has the responsibility of collecting orders and
-handing them to the solvers.
+The app uses a Wallet Connect v1 bridge.
```

Comment:
Not anymore. We have removed it, and this section should be removed as well

Reply:
fixed